src/com/theironyard/novauc/Customer.java

```diff
@@ -0,0 +1,149 @@
+        String answer = ATM2.nextLine();
+        if(answer.equalsIgnoreCase("yes")) {
+            actions.add(action);
+            System.out.println("This transaction has been recorded");
+        }
+
+    }
+
+    public double getBalance() {
+        return balance;
+    }
+    public void setBalance(double balance) {
+        this.balance = balance;
+    }
+
+public void chooseName() throws Exception {
+         HashMap<String, Double> bbustomers = new HashMap();
+         bbustomers.put("Andy", 500.00);
+         bbustomers.put("Becky", 600.00);
+         bbustomers.put("Beth", 650.00);
+         bbustomers.put("Dora", 750.00);
+         bbustomers.put("Elizabeth", 800.00);
+         bbustomers.put("Frank", 900.00);
+         bbustomers.put("Gary", 775.00);
+         bbustomers.put("Harry", 820.00);
+         bbustomers.put("Jack", 940.00);
+         bbustomers.put("Laura", 1000.00);
+         bbustomers.put("Maddy", 1150.00);
+         bbustomers.put("Peter", 860.00);
+         bbustomers.put("Steven", 770.00);
+         bbustomers.put("Tucker", 1200.00);
+         bbustomers.put("New Customer", 1250.00);
+
+    System.out.println("What is your name? (press Enter after entering your name)");
+    name = ATM2.scanner.nextLine();
+
+        if (bbustomers.containsKey(name)) {
+        System.out.println("Welcome! " + name + ", your balance is $" + bbustomers.get(name));
+        balance = bbustomers.get(name);
+
+    } else {
+        System.out.println("You do not have an account with our bank, " + name);
+        System.out.println("Please type your name exactly as you would like it to appear on your account below and press Enter");
+        name = ATM2.scanner.nextLine();
+        System.out.println("How much money would you like to add to your account " + name + "? (example: 200.00, 500.00)");
+        balance = ATM2.scannerDouble.nextDouble();
+        bbustomers.put(name, balance);
+
+    }
+}
+
+public void chooseRemove() {
+    HashMap<String, Double> bbustomers = new HashMap();
+    bbustomers.put("Andy", 500.00);
+    bbustomers.put("Becky", 600.00);
+    bbustomers.put("Beth", 650.00);
+    bbustomers.put("Dora", 750.00);
+    bbustomers.put("Elizabeth", 800.00);
+    bbustomers.put("Frank", 900.00);
+    bbustomers.put("Gary", 775.00);
+    bbustomers.put("Harry", 820.00);
+    bbustomers.put("Jack", 940.00);
+    bbustomers.put("Laura", 1000.00);
+    bbustomers.put("Maddy", 1150.00);
+    bbustomers.put("Peter", 860.00);
+    bbustomers.put("Steven", 770.00);
+    bbustomers.put("Tucker", 1200.00);
+    bbustomers.put("New Customer", 1250.00);
+
+//    System.out.println("Please type in your name again");
+
+    System.out.println(name + ", would you like to remove your account? (yes or no)");
+    String remove = ATM2.scanner.nextLine();
+    if (remove.equalsIgnoreCase("yes")) {
+        bbustomers.remove(name, balance);
```

Comment:
The way you have this set up it is not accessing the correct hashmap to remove accounts from.  You would need to move your hashmap to a class variable so that it can be accessed throughout the entire class, just like string name is.